server/src/main/java/org/elasticsearch/script/AbstractSortScript.java

```diff
@@ -55,8 +56,12 @@
      */
     private final LeafSearchLookup leafLookup;
 
+    /** Access fields **/
```

Comment:
Would you please either make this comment more descriptive or remove it.

Reply:
Removed.